pkg/kubelet/kubelet.go

```diff
@@ -2114,7 +2112,10 @@ func (kl *Kubelet) SyncPod(ctx context.Context, updateType kubetypes.SyncPodType
 		for _, r := range result.SyncResults {
 			if r.Action == kubecontainer.ResizePodInPlace && r.Error != nil {
 				// If the condition already exists, the observedGeneration does not get updated.
-				kl.statusManager.SetPodResizeInProgressCondition(pod.UID, v1.PodReasonError, r.Message, pod.Generation)
+				if kl.statusManager.SetPodResizeInProgressCondition(pod.UID, v1.PodReasonError, r.Message, pod.Generation) {
+					msg := events.PodResizeInProgressErrorMsg(pod, pod.Generation, r.Message)
```

Comment:
`pod.Generation` here is the "desired" generation, right? I think it should be the allocated generation for in-progress messages.